src/components/Chart/AreaChart.js

```diff
@@ -0,0 +1,31 @@
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+
+import { c3ChartDefaults } from './utils';
+
+const CHART_TYPE = 'area';
```

Comment:
I think we agreed to keep component level constants in a constants.js file in Avi's previous consistency updates...
example:
https://github.com/patternfly/patternfly-react/blob/master/src/components/ToastNotification/constants.js

Reply:
👍